Mac-App/Mac-App/AppDelegate.swift

```diff
@@ -36,3 +38,16 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
         backendSubscription.stopListening()
     }
 }
+
+// MARK: Akward macOS menu actions plugging
+
+/// Actions from menus will be forwarded to a designated data type to avoid unnecessary clutter in the
+/// AppDelegate.
+/// This mechanism (linking actions from `Main.storyboard`) is very strange but attempting to do this
+/// only with code is surprinsigly complicated. WTF Apple?
+
+extension AppDelegate {
+    @IBAction func triggerNewSearch(_ sender: NSMenuItem) {
+        macoOSMenu.triggerNewSearch(sender)
+    }
+}
```

Comment:
Interesting 🤔 @Thurman1776-

Reply:
Dude, this is horrible. It is really counter-intuitive. The more I do Mac development the more I sympathise the poor Xcode folks 😛